core/server/helpers/ghost_head.js

```diff
@@ -171,6 +179,10 @@ module.exports = function ghost_head(options) { // eslint-disable-line camelcase
                 if (client && client.id && client.secret && !_.includes(context, 'amp')) {
                     head.push(getAjaxHelper(client.id, client.secret));
                 }
+
+                if (!_.includes(context, 'amp') && labs.isSet('members')) {
```

Comment:
Why do we need the `amp` context condition for pushing members script? 🤔

Reply:
This is checking we don't have the AMP condition, and it's so that we don't attempt to load members for AMP output